pkg/js/parse_tests/domain-ip-map.json

```diff
@@ -0,0 +1,4 @@
+{
+  "foo.com": "1.2.3.4",
+  "bar.com": "1.2.3.5"
```

Comment:
Please make the two values very different (like... 1.1.1.1 and 5.5.5.5) so that visually they are very different.